bamboolean/lexer.py

```diff
@@ -105,65 +107,51 @@ def _integer(self):
             self.next()
         return int(result)
 
+    def skip_and_invoke(self, count, func):
```

Comment:
Not a big deal, but I don't see why this function helps at all. X_and_Y function is a smell, you could just have a function X and a function Y...

Reply:
When you take a look at `regex_map`, you will see that i can't invoke two functions in one lambda expression, that's why i wrote this function.

But then i may move skipping part to string() function that tokenize the string, and avoid having this function at all. Applied, please check.